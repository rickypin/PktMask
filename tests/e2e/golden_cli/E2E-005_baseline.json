{
  "test_id": "E2E-005",
  "name": "Dedup + Mask",
  "input_file": "tests/data/tls/tls_1_2-2.pcap",
  "input_hash": "baf39e40187111119c0de19358b3c944a07489806231fd24e6ab922b678b3ded",
  "cli_args": [
    "--dedup",
    "--mask"
  ],
  "output_hash": "f638af03eb9dfae58fbf5a8757c2c0b0947bfa2803687d9e18f2ec92ed3b5b03",
  "timestamp": "2025-10-09T21:31:10.596934",
  "version": "1.0-cli",
  "note": "Generated using CLI only - completely decoupled from API"
}