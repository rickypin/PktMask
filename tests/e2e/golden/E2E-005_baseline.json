{
  "test_id": "E2E-005",
  "name": "Dedup + Mask",
  "input_file": "tests/data/tls/tls_1_2-2.pcap",
  "input_hash": "baf39e40187111119c0de19358b3c944a07489806231fd24e6ab922b678b3ded",
  "config": {
    "dedup": true,
    "anon": false,
    "mask": true
  },
  "output_hash": "f638af03eb9dfae58fbf5a8757c2c0b0947bfa2803687d9e18f2ec92ed3b5b03",
  "output_file_size": 2721,
  "packet_count": 14,
  "stats": {
    "packets_processed": 14,
    "packets_modified": 1,
    "duration_ms": 427.5820255279541,
    "stages": [
      {
        "name": "DeduplicationStage",
        "packets_processed": 14,
        "packets_modified": 0,
        "duration_ms": 2.457141876220703
      },
      {
        "name": "Mask Payloads",
        "packets_processed": 14,
        "packets_modified": 1,
        "duration_ms": 6.543159484863281
      }
    ]
  },
  "timestamp": "2025-10-09T19:39:46.027327",
  "version": "1.0"
}